std/uuid.d

```diff
@@ -1085,8 +1088,14 @@ unittest
     return u;
 }
 
+private @trusted
+ref T unsafeTrustingArrayElementReinterpret(T)(ref ubyte[16] arr, size_t i)
+{
+    return *cast(T*)(arr.ptr+i);
```

Comment:
Unfortunately, that's precisely the usage that Walter and Andrei strongly disagree with. You'll have to take it up with them, since they have made it clear that we are not to merge anything that abuses `@trusted` in this way.